draft-irtf-cfrg-hpke.md

```diff
@@ -151,14 +151,19 @@ operations, roles, and behaviors of HPKE:
 - Ephemeral (E): A fresh random value meant for one-time use.
 - `(skX, pkX)`: A KEM key pair used in role X; `skX` is the private
   key and `pkX` is the public key
-- `pk(sk)`: The public key corresponding to a private key
-- `len(x)`: The two-octet length of the octet string `x`, in network
-  (big-endian) byte order
+- `pk(skX)`: The public key corresponding to a private key
+- `len(x)`: The length of the octet string `x`, expressed as a
+  two-octet unsigned integer in network (big-endian) byte order
 - `encode_big_endian(x, n)`: An octet string encoding the integer
   value `x` as an n-byte big-endian value
-- `+`: Concatenation of octet strings; `0x01 + 0x02 = 0x0102`
-- `*`: Repetition of an octet string; `0x01 * 4 = 0x01010101`
-- `^`: XOR of octet strings; `0xF0F0 ^ 0x1234 = 0xE2C4`
+- `concat(x0, ..., xN)`: Concatenation of octet strings.  An empty
+  value is treated as equivalent to an empty octet string.
+  `concat(0x01, None, 0x0203) = 0x010203`
```

Comment:
Why is this needed?

Reply:
You mean the "empty value" part?  I thought it was needed for #2, but on further reflection, it need not be.

Reply:
👍 I'd remove it if not used.